cms/src/components/languages/CreateOrEditLanguageModel.spec.ts

```diff
@@ -0,0 +1,113 @@
+import "fake-indexeddb/auto";
+import { describe, it, expect, beforeEach, afterEach } from "vitest";
+import { mount } from "@vue/test-utils";
+import CreateOrEditLanguageModal from "./CreateOrEditLanguageModal.vue";
+import { setActivePinia } from "pinia";
+import { createTestingPinia } from "@pinia/testing";
+import { accessMap, db } from "luminary-shared";
+import {
+    fullAccessToAllContentMap,
+    mockLanguageDtoEng,
+    mockLanguageDtoFra,
+    mockLanguageDtoSwa,
+} from "@/tests/mockdata";
+
+describe("CreateOrEditLanguageModal.vue", () => {
+    beforeEach(async () => {
+        await db.docs.bulkPut([mockLanguageDtoEng, mockLanguageDtoFra, mockLanguageDtoSwa]);
+
+        setActivePinia(createTestingPinia());
+
+        accessMap.value = fullAccessToAllContentMap;
+    });
+
+    afterEach(async () => {
+        // Clear the database after each test
+        await db.docs.clear();
+        await db.localChanges.clear();
+    });
+
+    describe("on create mode", () => {
```

Comment:
It is a bit confusing with the naming. Perhaps it should just read "Create mode"?